src/main/java/org/matsim/nemo/CellRelocator.java

```diff
@@ -0,0 +1,366 @@
+import java.util.logging.Logger;
+
+public class CellRelocator {
+    private static Logger logger = Logger.getLogger("CellRelocator");
+    public List<Coord> cells = new ArrayList<>();
+    Path relocationData;
+    private Population population;
+    private QuadTree quadTree;
+    private int limits = 16000;
+    private int relocatedEverything = 0;
+    private int relocatedOnlyHome = 0;
+    //Summation of these two ratios must be less than or equal to 1
+    final static double relocateEverythingFraction = 0; //eg. 0.4 means 40 percent of population is moving everything
+    final static double relocateOnlyHomeFraction = 1;  //eg. 0.3 means 30 percent of population is moving only home
+
+    /**
+     * Constructor
+     * @param relocationData Path to relocation file
+     * @param population     Population population
+     * @param outerGeometry  Outer geometry
+     */
+    CellRelocator(Path relocationData, Population population, Geometry outerGeometry) {
+        initializeCells();
+        initializeSpatialIndex(population, outerGeometry);
+        this.relocationData = relocationData;
+        this.population = population;
+    }
+
+    /**
+     * Creates a circle shape using geotools
+     * Link: http://docs.geotools.org/stable/userguide/library/jts/geometry.html
+     *
+     * @param coord  coordinates of home are passed in
+     * @param RADIUS radius value is passed in meters
+     * @return Circle shape as a geometry
+     */
+    private static Geometry createCircle(Coord coord, final double RADIUS) {
+        GeometricShapeFactory shapeFactory = new GeometricShapeFactory();
+        shapeFactory.setNumPoints(32);
+        shapeFactory.setCentre(new Coordinate(coord.getX(), coord.getY()));
+        shapeFactory.setSize(RADIUS * 2);
+        return shapeFactory.createCircle();
+    }
+
+    /**
+     * Getter of Population
+     * @return population variable
+     */
+    public Population getPopulation() {
+        return population;
+    }
+
+    /**
+     * Initializes the cells from the CSV file
+     */
+    private void initializeCells() {
+        String x;
+        String y;
+        Coord coord;
+        boolean firstLine = false;
+
+        Iterable<CSVRecord> records;
+        try (Reader in = new FileReader("/Users/nanddesai/Documents/mercator-nemo/src/relocationInput.csv")) {
+            records = CSVFormat.EXCEL.parse(in);
+            for (CSVRecord record : records) {
+                if (!firstLine) {
+                    firstLine = true;
+                } else if (record.iterator().hasNext()) {
+                    x = record.get(1);
+                    y = record.get(2);
+                    coord = new Coord(Double.parseDouble(x), Double.parseDouble(y));
+                    cells.add(coord);
+                } else {
+                    break;
+                }
+            }
+            logger.info("CSV Coordinates File has been parsed.");
+        } catch (FileNotFoundException e) {
```

Comment:
Please catch both exceptions in one block and rethrow a runtime exception like this:

catch(FileNotFoundException | IOException e) {
...
}